### DISCUSSION TITLE
Horizontal scrolling using phone with Apollo Template

### DISCUSSION THREAD

**Discussion by egarciaf2** [primefaces/community#1738] [PrimeNG Templates] [answered]
In the apollo angular template, in the table view there is an error that generates an orizontal scroll, which is uncomfortable, this is present in the demo that you have available, I attach screenshots, so you can support in solving the problem, only occurs in mobiles.

![apollo primeng org_apps_kanban(Samsung Galaxy S20 Ultra) (1)](https://github.com/primefaces/community/assets/46494114/a8de39c7-38ea-4091-aead-677f72d9d800)
![apollo primeng org_apps_kanban(Samsung Galaxy S20 Ultra)](https://github.com/primefaces/community/assets/46494114/59c8125c-8e15-487e-b782-e4585dc82c3c)

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for reporting the issue, we'll investigate it.

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, you can fix the issue by updating your PrimeNG version to the latest version.
